webpack/assets/javascripts/react_app/components/ReportTemplate/Inputs/ReportDateTime.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DateTimePicker, FieldLevelHelp } from 'patternfly-react';
+
+import Form from '../../common/forms/CommonForm';
+import { documentLocale } from '../../../common/I18n';
+import './DateTimeOverrides.scss';
+
+const ReportDateTime = ({ label, id, info, isRequired, locale }) => {
+  const currentLocale = locale || documentLocale();
+
+  return (
+    <Form
+      label={label}
+      required={isRequired}
+      tooltipHelp={info && <FieldLevelHelp content={<div>{info}</div>} />}
```

Comment:
is it possible to use `React.Fragment` instead of a `div`, the DOM would be cleaner